dist/stat_data.py

```diff
@@ -403,8 +403,10 @@ def __init__(self, name, desc, flags=''):
     CheckpointStat('checkpoint_time_recent', 'most recent time (msecs)', 'no_clear,no_scale'),
     CheckpointStat('checkpoint_time_total', 'total time (msecs)', 'no_clear,no_scale'),
     CheckpointStat('checkpoint_wait_reduce_dirty', 'wait cycles while cache dirty level is decreasing'),
-    CheckpointStat('checkpoints', 'number of checkpoints started'),
+    CheckpointStat('checkpoints_session', 'number of checkpoints started by session'),
```

Comment:
Would better to rename this to checkpoints_api. Checkpoints are all started by session. The difference is whether it is an internal session or a user session.